aviary/interface/test/test_basic_report.py

```diff
@@ -2,16 +2,15 @@
 from pathlib import Path
 import unittest
 import warnings
-
-from openmdao.utils.testing_utils import use_tempdirs
+from openmdao.utils.testing_utils import use_tempdirs, set_env_vars
+import openmdao.api as om
 
 from aviary.interface.default_phase_info.height_energy import phase_info
 from aviary.interface.methods_for_level1 import run_aviary
 from aviary.variable_info.enums import Verbosity
 
 
 @unittest.skip("Skipping test due to sensitivity in setup. Need tolerances on actual values")
```

Comment:
Do we have an issue out to come back to this later?

Reply:
Bret has added some utilities for comparing text stings of numerical values subject to tolerances: 

https://github.com/OpenMDAO/OpenMDAO/blob/master/openmdao/utils/testing_utils.py#L401